packages/core/src/app/shipping/Shipping.tsx

```diff
@@ -219,6 +223,18 @@ class Shipping extends Component<ShippingProps & WithCheckoutShippingProps, Ship
             }
         }
 
+        if (!isMultiShippingMode && isNewMultiShippingUIEnabled) {
+            this.setState({ isInitializing: true });
+
+            try {
+                await deleteConsignments();
+            } catch (error) {
+                onUnhandledError(error);
+            } finally {
+                this.setState({ isInitializing: false });
```

Comment:
I see these four lines are identical to the switching to single shipping case.

Perhaps we can place the `if` condition inside the `try` block to minimize code repetition.